docs/tutorials/understanding_gui/step_02/ReadMe.md

```diff
@@ -0,0 +1,64 @@
+> You can download the code of this step [here](../src/step_02.py) or all the steps [here](https://github.com/Avaiga/taipy-getting-started-gui/tree/develop/src).
+
+!!! warning "For Notebooks"
+
+    The "Getting Started" Notebook is available [here](https://docs.taipy.io/en/latest/getting_started/getting-started-gui/getting_started.ipynb). In Taipy GUI, the process to execute a Jupyter Notebook is different from executing a Python Script.
+
+# Step 2: Visual elements
+
+Many visual elements can be added to the basic code viewed in Step 1. This Step shows how to use visual elements 
+like charts, sliders, and tables, atc., inside the graphical interface.
+
+## Visual elements
+
+Taipy GUI can be considered as an **augmented** Markdown; it adds the concept of 
+**[Visual elements](https://docs.taipy.io/en/latest/manuals/gui/viselements/)** on top of all the Markdown syntax. A visual 
+element is a Taipy graphical object displayed on the client. It can be a 
+[slider](https://docs.taipy.io/en/latest/manuals/gui/viselements/slider/), a 
+[chart](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/), a 
+[table](https://docs.taipy.io/en/latest/manuals/gui/viselements/table/), an 
+[input](https://docs.taipy.io/en/latest/manuals/gui/viselements/input/), a 
+[menu](https://docs.taipy.io/en/latest/manuals/gui/viselements/menu/), etc. Check the list 
+[here](https://docs.taipy.io/en/latest/manuals/gui/controls/).
```

Comment:
Broken link here: https://docs.taipy.io/en/latest/manuals/gui/viselements/controls/

Reply:
Thx. I did not change the content of the old GS at all, and all the links must be updated anyway (we must have relative paths and not absolute paths anymore)
I propose not to fix the link in this PR.
We need to synchronize on how to turn the Getting Started into a tutorial. The content should be changed slightly...